components/elm/src/biogeophys/BalanceCheckMod.F90

```diff
@@ -619,7 +619,10 @@ subroutine ColWaterBalanceCheck( bounds, num_do_smb_c, filter_do_smb_c, &
        found = .false.
        do p = bounds%begp, bounds%endp
           if (veg_pp%active(p)) then
-             if ( (errsol(p) /= spval) .and. (abs(errsol(p)) > 1.e-7_r8) ) then
+             !if ( (errsol(p) /= spval) .and. (abs(errsol(p)) > 1.e-7_r8) ) then
+             ! solar radiation balance error is high when running FATES
+             ! adjust the threshold to 5.e-7
+             if ( (errsol(p) /= spval) .and. (abs(errsol(p)) > 5.e-7_r8) ) then
```

Comment:
I like the addition of this change, but we should use the is_fates() filter.  If the filter is true, then increase the error, if not, use the existing (stricter)

https://github.com/E3SM-Project/E3SM/blob/master/components/elm/src/data_types/VegetationType.F90#L71

Reply:
Updated this with the recommended check by setting an `errtol` local variable to be the lower value by default unless veg type is fates.